TileStache/Vector/__init__.py

```diff
@@ -539,7 +540,7 @@ class Provider:
         See module documentation for explanation of constructor arguments.
     """
     
-    def __init__(self, layer, driver, parameters, clipped, verbose, projected, spacing, properties, precision, id_property):
+    def __init__(self, layer, driver, parameters, clipped, verbose, projected, spacing, properties, precision, id_property, empty_fields):
```

Comment:
Make this parameter optional (ie `empty_fields=True`), to keep backward compatibility